pyproject.toml

```diff
@@ -2,33 +2,31 @@
 name = "nominal"
 version = "1.0.0-beta"
 description = "Automate Nominal workflows in Python"
-authors = ["JP <jack@nominal.io>", "Michael Bauer <michael@nominal.io>", "Alexander Reynolds <alex.reynolds@nominal.io>"]
-readme = "README.md"
-packages = [
-    {include = "nominal/py.typed"},
+authors = [
+    "JP <jack@nominal.io>",
+    "Michael Bauer <michael@nominal.io>",
+    "Alexander Reynolds <alex.reynolds@nominal.io>",
 ]
+maintainers = ["Alexander Reynolds <alex.reynolds@nominal.io>"]
+readme = "README.md"
+license = "MIT"
+homepage = "https://nominal.io"
+repository = "https://github.com/nominal-io/nominal-client"
 include = ["README.md", "LICENSE"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
 requests = "^2.32.3"
 python-dateutil = "^2.9.0.post0"
-jsondiff = "^2.2.0"
 polars = "^1.4.1"
-keyring = "^25.0.0"
-rich = "^13.7.1"
 conjure-python-client = ">=2.8.0,<3"
 pandas = ">=2.2,<3.0"
 pydantic = "^2.9.1"
 typing-extensions = "^4.12.2"
 click = "^8.1.7"
 
-[tool.poetry.dev-dependencies]
-isort = "^5.13.2"
-ruff = "^0.4.2"
-
```

Comment:
cool I made these changes too so we should centralise on this PR - I'll push the change in a sec